src/core/view/overlays/BaseStrokeToolView.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Xournal++
+ *
+ * Virtual class for showing overlays (e.g. active tools, selections and so on)
+ *
+ * @author Xournal++ Team
+ * https://github.com/xournalpp/xournalpp
+ *
+ * @license GNU GPLv2 or later
+ */
+
+#pragma once
+
+#include <cairo.h>
+
+#include "model/LineStyle.h"
+#include "util/Color.h"
+#include "view/overlays/OverlayView.h"
+
+class Stroke;
+
+namespace xoj::view {
+class Repaintable;
+
+class BaseStrokeToolView: public OverlayView {
```

Comment:
Needs a doc comment.

What are intended subclasses of `BaseStrokeToolView` other than the `BaseShapeOrSpineToolView` family?

Reply:
There will be StrokeToolView, in #4158